internal/services/mysql/mysql_flexible_server_resource_test.go

```diff
@@ -989,6 +990,7 @@ resource "azurerm_mysql_flexible_server" "test" {
     size_gb           = %d
     iops              = %d
     auto_grow_enabled = %t
+	iops_auto_scaling = %t
```

Comment:
This isn't lined up properly 
```suggestion
    iops_auto_scaling = %t
```